packages/node-sdk/paima-utils-backend/src/cde-access-internals.ts

```diff
@@ -229,3 +235,22 @@ export async function internalGetCardanoProjectedNft(
 
   return results;
 }
+
+export async function internalGetCardanoAssetUtxos(
+  readonlyDBConn: Pool,
+  address: string,
+  filterBy: string,
```

Comment:
I think you can replace this by `filterBy: keyof ICdeCardanoAssetUtxosByAddressResult` or just `filterBy: 'policy_id' | 'cip14_fingerprint'`